Description: rename executable to be sure it stay unique
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2025-02-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.1)
 set(CMAKE_CONFIGURATION_TYPES "Debug;Release")
-project(region_de_magallanes)
+project(regiondemagallanes-by-faemiyah)
 
 enable_language(C)
 enable_language(CXX)
@@ -19,7 +19,7 @@
 
 include_directories("${PROJECT_SOURCE_DIR}/src")
 
-add_executable(region_de_magallanes
+add_executable(regiondemagallanes-by-faemiyah
   "src/bsd_rand.c"
   "src/bsd_rand.h"
   "src/glsl_program.cpp"
@@ -32,22 +32,22 @@
   "src/image_png.hpp"
   "src/intro.cpp")
 if(MSVC)
-  target_link_libraries(region_de_magallanes "Imm32")
-  target_link_libraries(region_de_magallanes "Version")
-  target_link_libraries(region_de_magallanes "WINMM")
-  target_link_libraries(region_de_magallanes "${OPENGL_gl_LIBRARY}")
-  target_link_libraries(region_de_magallanes debug "${GLEW_LIBRARY_DEBUG}" optimized "${GLEW_LIBRARY}")
-  target_link_libraries(region_de_magallanes debug "${PNG_LIBRARY_DEBUG}" optimized "${PNG_LIBRARY}")
-  target_link_libraries(region_de_magallanes debug "${SDL2_LIBRARY_DEBUG}" optimized "${SDL2_LIBRARY}")
-  target_link_libraries(region_de_magallanes debug "${ZLIB_LIBRARY_DEBUG}" optimized "${ZLIB_LIBRARY}")
+  target_link_libraries(regiondemagallanes-by-faemiyah "Imm32")
+  target_link_libraries(regiondemagallanes-by-faemiyah "Version")
+  target_link_libraries(regiondemagallanes-by-faemiyah "WINMM")
+  target_link_libraries(regiondemagallanes-by-faemiyah "${OPENGL_gl_LIBRARY}")
+  target_link_libraries(regiondemagallanes-by-faemiyah debug "${GLEW_LIBRARY_DEBUG}" optimized "${GLEW_LIBRARY}")
+  target_link_libraries(regiondemagallanes-by-faemiyah debug "${PNG_LIBRARY_DEBUG}" optimized "${PNG_LIBRARY}")
+  target_link_libraries(regiondemagallanes-by-faemiyah debug "${SDL2_LIBRARY_DEBUG}" optimized "${SDL2_LIBRARY}")
+  target_link_libraries(regiondemagallanes-by-faemiyah debug "${ZLIB_LIBRARY_DEBUG}" optimized "${ZLIB_LIBRARY}")
 else()
-  target_link_libraries(region_de_magallanes "${BOOST_FILESYSTEM_LIBRARY}")
-  target_link_libraries(region_de_magallanes "${BOOST_PROGRAM_OPTIONS_LIBRARY}")
-  target_link_libraries(region_de_magallanes "${BOOST_SYSTEM_LIBRARY}")
-  target_link_libraries(region_de_magallanes "${BOOST_THREAD_LIBRARY}")
-  target_link_libraries(region_de_magallanes "${BOOST_WAVE_LIBRARY}")
-  target_link_libraries(region_de_magallanes "${GLEW_LIBRARY}")
-  target_link_libraries(region_de_magallanes "${PNG_LIBRARY}")
-  target_link_libraries(region_de_magallanes "${SDL2_LIBRARY}")
+  target_link_libraries(regiondemagallanes-by-faemiyah "${BOOST_FILESYSTEM_LIBRARY}")
+  target_link_libraries(regiondemagallanes-by-faemiyah "${BOOST_PROGRAM_OPTIONS_LIBRARY}")
+  target_link_libraries(regiondemagallanes-by-faemiyah "${BOOST_SYSTEM_LIBRARY}")
+  target_link_libraries(regiondemagallanes-by-faemiyah "${BOOST_THREAD_LIBRARY}")
+  target_link_libraries(regiondemagallanes-by-faemiyah "${BOOST_WAVE_LIBRARY}")
+  target_link_libraries(regiondemagallanes-by-faemiyah "${GLEW_LIBRARY}")
+  target_link_libraries(regiondemagallanes-by-faemiyah "${PNG_LIBRARY}")
+  target_link_libraries(regiondemagallanes-by-faemiyah "${SDL2_LIBRARY}")
 endif()
-target_link_libraries(region_de_magallanes "${THREADS_LIBRARY}")
+target_link_libraries(regiondemagallanes-by-faemiyah "${THREADS_LIBRARY}")
